sentry/src/test/java/io/sentry/SentryClientTest.kt

```diff
@@ -328,6 +337,52 @@ class SentryClientTest {
         assertEquals(allEvents, mockingDetails(fixture.connection).invocations.size - 1) // 1 extra invocation outside .send()
     }
 
+    @Test
+    fun `when captureUserFeedback, envelope is sent`() {
+        val sut = fixture.getSut()
+
+        sut.captureUserFeedback(userFeedback)
+
+        val captor = ArgumentCaptor.forClass(SentryEnvelope::class.java)
+        verify(fixture.connection).send(captor.capture())
+
+        val actual = captor.value
+        assertEquals(userFeedback.eventId, actual.header.eventId)
+        assertEquals(fixture.sentryOptions.sdkVersion, actual.header.sdkVersion)
+
+        assertEquals(1, actual.items.count())
+        val item = actual.items.first()
+        assertEquals(SentryItemType.User_Report, item.header.type)
+        assertEquals("application/json", item.header.contentType)
+
+        assertEnvelopeItemDataForUserFeedback(item)
+    }
+
+    private fun assertEnvelopeItemDataForUserFeedback(item: SentryEnvelopeItem) {
+        val stream = ByteArrayOutputStream()
+        val writer = BufferedWriter(OutputStreamWriter(stream, Charset.forName("UTF-8")))
```

Comment:
M: kotlin has nice extension methods for that, we could use them, look for `bufferedReader` or similar ones